src/routes/+page.svelte

```diff
@@ -1,2 +1,108 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://svelte.dev/docs/kit">svelte.dev/docs/kit</a> to read the documentation</p>
+<script lang="ts">
+	let { data } = $props();
+
+	interface Section {
+		key: string;
+		dynamic: string;
+		el: HTMLElement | undefined;
+		value: number;
+	}
+
+	const sections: Section[] = $state(
+		data.dynamics.map((dynamic, idx) => {
+			return {
+				key: 'ABCDEFGH'[idx],
+				dynamic,
+				el: undefined,
+				value: 3
+			};
+		})
+	);
+
+	function scrollToSection(evt: Event, idx: number) {
+		// supercedes the default a tag link for smooth scrolling if javascript is enabled
+		evt.preventDefault();
+		const { el, key } = sections[idx];
+		if (!el) return;
+		el.scrollIntoView({
+			behavior: 'smooth'
+		});
+		// focus the input in the given section once in view
+		document.getElementById(key)?.focus();
```

Comment:
The main motivation was enabling the page the scroll smoothly instead of jumping to the next section, and makes the next/prev buttons work without javascript. But without javascript the values won't update the final link so it would only look like it's working. I'm open to buttons here too!